links/src/commonTest/kotlin/handler/PlatformLinksHandlerTest.kt

```diff
@@ -57,3 +52,19 @@ class IosLinksHandlerTest {
         assertEquals(emptyList(), linksValidator.extractQueryAsList(url))
     }
 }
+
+object TestConstants {
+
+    val VALID_URLS = listOf(
+        "https://test.io",
+        "http://test.io",
+        "https://test.io?isValid=true",
+        "https://test.io?list_1=first&list_2=second&list_3=third"
+    )
+
+    val INVALID_URLS = listOf(
+        "notvalid.com",
+        "not valid",
+        "httpss://notvalid.com?/?isValid=false"
```

Comment:
Hm, I thought custom scheme is valid (e.g. `myapp://doit/`)

Reply:
Apparently not in the current implementation. Perhaps @corrado4eyes can shine a light on that.
We validate using `NSURLConnection.canHandleRequest(NSURLRequest.requestWithURL(nsUrl))` which at least in tests fails if I provide a custom scheme

Reply:
Reading up on it, custom schemes are not supported for deeplinks on iOS anymore. So Im leaving it be for now

Reply:
Thanks, good to know

Reply:
@avdyushin Good point. Right now custom scheme are not supported, when I made the module I thought it mainly for App links and Universal links. An improvement could be add a layer of abstraction on top of this where its possible to specify what kind of link you are expecting (or a list of `LinksTypes` or whatever). What do you think?

As far as I know the preferred way is of course App links and Universal links, but I can still see apps that uses schemes especially on iOS, on Android less because there is the annoying "Open app with" so more and more apps are migrating to App links.

Reply:
I'm find to keep as is Universal Links is future